### DISCUSSION TITLE
Show "true" lat/lng (rather then reverse geocode)

### DISCUSSION THREAD

**Discussion by Squiglygon** [photoprism/photoprism#1485] [Q&A]
Good day.

I'm using photoprism to index marine mammals surveys conducted at sea.
Currently, if I want to use the map feature, I must also use the reverse geocoding which results in some of the images being tagged as the closest city or other places.
This happens with other images too.

Is there a way to use the map feature, but without reverse geo-coding? just use the exact lat/lng of each image?

**Comment by lastzero** [comment]
No, that's a special use case we didn't consider so far. We were expecting that users who don't use geocoding also don't use maps as their ability to peform searches would be very limited. PhotoPrism groups exact coordinates by place based on geo metadata. If that isn't available, all that's left is in fact showing all photos on a map without search (you could still zoom in to specific coordinates as a simple form of search).

**Reply by Squiglygon** [reply]
Is it actually possible? showing all photos on a map based on coords?
I do find a feature where you can show all photos (given simple criteria like dates) on a map very useful.
E.g (regardless of my specific case) for people with photos in remote areas (think hiking, boating, endurance tracks, etc) which aren't being geo-coded all that great.
